src/ContentCreator.php

```diff
@@ -718,6 +719,8 @@ protected function getRandomWeighted(array $list) {
    *   Flag to keep CSV files after import.
    */
   public function importCsvFiles($keep_content_files = FALSE) {
+    $this->storage->startContentCreationProcess();
```

Comment:
This should only occur in the child processes. So after the fork.

Reply:
I did that first. But then the process is still failing

Reply:
That's weird. How does it fail?

Reply:
The terminate event still occurs twice at the end, which executes the RequestCloseSubscriber, which wants to clear the cache, which ends in a db lock.

Reply:
You got to wonder why...

Does this happen even when you change the cache clear in the Drush command to drupal_flush_all_caches()?

Reply:
With drupal_flush_all_caches I got even more problems. Then funnily the inContentCreationProcess var was never set in the child processes. Strange stuff.